tslint.json

```diff
@@ -1,36 +1,39 @@
 {
-    "defaultSeverity": "error",
-    "extends": [
-      "typestrict",
-      "tslint:recommended",
-      "tslint-immutable",
-      "tslint-sonarts",
-      "tslint-plugin-prettier",
-      "tslint-config-prettier"
+  "defaultSeverity": "error",
+  "extends": [
+    "typestrict",
+    "tslint:recommended",
+    "tslint-immutable",
+    "tslint-sonarts",
+    "tslint-plugin-prettier",
+    "tslint-config-prettier"
+  ],
+  "jsRules": {},
+  "rules": {
+    "no-null-keyword": true,
+    "no-any": true,
+    "no-var-keyword": true,
+    "typedef": [
+      true,
+      "call-signature"
     ],
-    "jsRules": {},
-    "rules": {
-      // Disallows use of the null keyword literal.
-      "no-null-keyword": true,
-      // Disallows usages of any as a type declaration.
-      "no-any": true,
-
-      // tslint-immutable: Recommended built-in rules
-      "no-var-keyword": true,
-      "typedef": [true, "call-signature"],
-
-      // tslint-immutable: Immutability rules
-      "readonly-keyword": true,
-      "readonly-array": true,
-      "no-let": true,
-      "no-object-mutation": true,
-      "no-method-signature": true,
-
-      // tslint-immutable: Functional style rules
-      // "no-this": true,
-      // "no-class": true,
-      "no-mixed-interface": false,
-      "prettier": true
-    },
-    "rulesDirectory": []
-}
+    "readonly-keyword": true,
+    "readonly-array": true,
+    "no-let": true,
+    "no-method-signature": true,
+    "no-mixed-interface": false,
+    "prettier": true,
+    "no-console": false,
+    "no-object-mutation": [
+      true,
+      {
+        "ignore-prefix": [
```

Comment:
it doesn't look like there's any object mutation in this code, you can remove this exception